apps/test/unit/gamelab/spritelab/spriteUtilsTest.js

```diff
@@ -208,4 +209,223 @@ describe('Sprite Utils', () => {
       ]);
     });
   });
+
+  describe('Events', () => {
+    it('Can run key events', () => {
+      const keyWentDownStub = stub(gameLabP5.p5, 'keyWentDown').returns(true);
+      const keyDownStub = stub(gameLabP5.p5, 'keyDown').returns(true);
+      const eventLog = [];
+      spriteUtils.addEvent('whenpress', {key: 'up'}, () =>
+        eventLog.push('when up press ran')
+      );
+      spriteUtils.addEvent('whilepress', {key: 'down'}, () =>
+        eventLog.push('while down press ran')
+      );
+
+      spriteUtils.runEvents(gameLabP5.p5);
+
+      expect(eventLog).to.deep.equal([
+        'when up press ran',
+        'while down press ran'
+      ]);
+
+      keyWentDownStub.restore();
+      keyDownStub.restore();
+    });
+
+    describe('Click events', () => {
+      let eventLog, mouseWentDownStub, mouseIsOverStub, mousePressedOverStub;
+      beforeEach(function() {
+        eventLog = [];
+        mouseWentDownStub = stub(gameLabP5.p5, 'mouseWentDown');
+        mouseIsOverStub = stub(gameLabP5.p5, 'mouseIsOver');
+        mousePressedOverStub = stub(gameLabP5.p5, 'mousePressedOver');
+      });
+      afterEach(function() {
+        mouseWentDownStub.restore();
+        mouseIsOverStub.restore();
+        mousePressedOverStub.restore();
+      });
+      it('works with individual sprites', () => {
```

Comment:
NIT: 'work with individual sprites'

Reply:
Same with others below

Reply:
Why "work"? I was trying to make it read like "it works with..."